api/worker.go

```diff
@@ -15,6 +15,10 @@ import (
 )
 
 var (
+	// ErrBucketMissing is returned by the worker API by endpoints that need a
+	// bucket when it wasn't specified.
+	ErrBucketMissing = errors.New("bucket parameter not specified")
```

Comment:
`nit` we should be more consistent and do `ErrBucketRequired = errors.New("'bucket' parameter is required")` where we use x is required and make sure the parameter name is between quotes. Including this one we have three variations of this.